### PR TITLE
New d2k build palette backup

### PR DESCRIPTION
**Pull request by xan2622**
- merged light and heavy vehicles
- chrome.png updated (without Atreides and Ordos buttons)
- slightly modified UI background with repeated texture
- Debug button (the gear) re-added in chrome.png

### CODE REVIEW

File: mods/d2k/rules/atreides.yaml [hunk 1]
```diff
@@ -172,7 +172,7 @@ COMBATA.starport:
 SONICTANK:
 	Inherits: ^Vehicle
 	Buildable:
-		Queue: Armor
+		Queue: Vehicle
```
Comment (Mailaender) [inline mods/d2k/rules/atreides.yaml:175]: Please don't mix controversial balancing changes with UI improvements here.

**Comment by Mailaender** [comment]
You broke the button backgrounds and left a gap at the armor queue:

image

**Comment by Mailaender** [comment]
I will try to fix things up myself based on your PNG changes. Please stand by.

File: mods/d2k/rules/atreides.yaml (hunk 1 above)
Comment (xan2622) [inline mods/d2k/rules/atreides.yaml:175]: I thought it was a request : merging light and heavy vehicles..
https://github.com/OpenRA/OpenRA/issues/6043#issuecomment-71596724

**Comment by xan2622** [comment]
I didn't notice the lack of button backgrounds in my own branch. 

PS : it is more likely to be caused by my lack of tests with Atreides and Ordos factions. Sorry. :-/ 
(I guess, it doesn't happen with Harkonnen)

**Comment by Mailaender** [comment]
Shall we go with a thin wooden border and more of the black background from the rest of the dialogs

image

or the new wooden grain:

image

File: mods/d2k/chrome/ingame-player.yaml [hunk 2]
```diff
@@ -278,7 +278,7 @@ Container@PLAYER_WIDGETS:
 							Children:
 								Image@ICON:
 									X: 5
-									Y: 4
+									Y: 5
 									ImageCollection: production-icons
 						ProductionTypeButton@VEHICLE:
 							Logic: AddRaceSuffixLogic
```
Comment (Mailaender) [inline mods/d2k/chrome/ingame-player.yaml:284]: I just noticed that these are in fact redundant now that we only have one artwork for all. Will fix it.

**Pull request closed without merging**